features/step_definitions/section_explicit_waiting_steps.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+When('I wait for the section that takes a while to appear') do
+  @test_site.home.slow_section(wait: 1)
+end
+
+When('I wait for the section that takes a while to vanish') do
+  @test_site.home.wait_until_vanishing_section_invisible
+end
+
+Then("an exception is raised when I wait for a section that won't appear") do
+  expect { @test_site.section_experiments.slow_section(wait: 0.05) }
+    .to raise_error(Capybara::ElementNotFound)
+end
+
+Then('an error is raised when waiting for the section to vanish') do
```

Comment:
an **Element Invisible Timeout** error is raised when waiting for the section to vanish